SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestLBDelegationInterface.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "SpatialFunctionalTestLBDelegationInterface.h"
+#include "GameFramework/Actor.h"
+#include "SpatialCommonTypes.h"
+#include "Utils/SpatialStatics.h"
+#include "SpatialFunctionalTestWorkerDelegationComponent.h"
+#include "EngineUtils.h"
+
+bool ISpatialFunctionalTestLBDelegationInterface::AddActorDelegation(AActor* Actor, VirtualWorkerId WorkerId, bool bPersistOnTestFinished /*= false*/)
+{
+	if (Actor == nullptr)
+	{
+		return false;
+	}
+
+	if (!Actor->HasAuthority())
+	{
+		ensureMsgf(false, TEXT("Only Authority of an Actor can delegate it to another worker. Tried to delegate %s to %d"), *Actor->GetName(), WorkerId);
```

Comment:
```suggestion
		ensureMsgf(false, TEXT("Only the worker authoritative over an Actor can delegate it to another worker. Tried to delegate %s to %d"), *Actor->GetName(), WorkerId);
```
Nit: just closer to our other logs in the GDK